WebRoot/js/zimbraAdmin/statistics/view/ZaGlobalAdvancedStatsPage.js

```diff
@@ -192,7 +192,7 @@ ZaGlobalAdvancedStatsPage.plotGlobalQuickChart = function (id, group, columns, c
                 for (var i = 0; i < timeStampLength; i++) {
                     var valuesSumAtTs = 0;
                     for (var j = 0; j < hostNameLen; j++) {
-                        if (hostname[j].stats[0].values[i] && hostname[j].stats[0].values[i].stat[0]) {
+                        if (hostname[j].stats && hostname[j].stats[0].values && hostname[j].stats[0].values[i] && hostname[j].stats[0].values[i].stat[0]) {
```

Comment:
Not sure we will have data like that but it is always safe to have proper checks

Fixed this